CMakePresets.json

```diff
@@ -22,7 +22,7 @@
       "cacheVariables": {
         "CMAKE_TOOLCHAIN_FILE": {
           "type": "STRING",
-          "value": "$env{VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake"
+          "value": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
```

Comment:
Please do not change this variable, this is used for the ci

Reply:
May I suggest updating [ExamplePlugin](https://github.com/Ryan-rsm-McKenzie/ExamplePlugin-CommonLibSSE) with `VCPKG_INSTALLATION_ROOT` for more consistency?